modules/ossm-threescale-templates.adoc

```diff
@@ -6,14 +6,25 @@
 [id="ossm-threescale-templates_{context}"]
 = Generate templates from URL examples
 
-* This example generates templates allowing the token, URL pair to be shared by multiple services as a single handler:
+[NOTE]
+====
+* Run the following commands via `oc exec` from the 3scale adapter container image in xref:ossm-threescale-manifests_{context}[Generating manifests from a deployed adapter].
+* Use the `3scale-config-gen` command to help avoid YAML syntax and indentation errors.
+====
+
+.Procedure
+
+* Use the `3scale-config-gen` command to autogenerate templates files allowing the token, URL pair to be shared by multiple services as a single handler:
 +
 ----
-$ 3scale-gen-config --name=admin-credentials --url="https://<organization>-admin.3scale.net:443" --token="[redacted]"
+$ 3scale-config-gen --name=admin-credentials --url="https://<organization>-admin.3scale.net:443" --token="[redacted]"
 ----
-
-* This example generates the templates with the service ID embedded in the handler:
++
+* The following example generates the templates with the service ID embedded in the handler:
 +
 ----
-$ 3scale-gen-config --url="https://<organization>-admin.3scale.net" --name="my-unique-id" --service="123456789" --token="[redacted]"
+$ 3scale-config-gen --url="https://<organization>-admin.3scale.net" --name="my-unique-id" --service="123456789" --token="[redacted]"
```

Comment:
I dont think you need to specify the --service and --token in config gen as it is now provided via annotations. @unleashed  Could you please confirm this?

Reply:
The `--token` is needed (as it will always be part of the handler), but you are right that `service` can be omitted if it is provided via the annotations - there should already be some section explaining (and encouraging the user) to use the annotations, though. Perhaps we can add a note next to this example to mention that you can omit the `service` if you will use the annotations. Btw, @dfennessy as you also missed the important detail that this should be invoked from within the container image via `oc exec`, both notes could be grouped together next to this example.